projects/miragesdk/examples/mirage-dhcp.yml

```diff
@@ -13,7 +13,7 @@ services:
   - name: rngd
     image: mobylinux/rngd:3dad6dd43270fa632ac031e99d1947f20b22eec9
   - name: dhcp-client
-    image: mobylinux/dhcp-client:a7a6b49b0ff51ffa2f44ac848cd649e29f946e0c
+    image: mobylinux/dhcp-client:24e63a65d9a7a20f0f9b3b10e86eeb07b3e26427
```

Comment:
cool, please give it to docker hub username `yomimono` rather than `mindypreston`